javascript/ql/test/query-tests/Security/CWE-754/UnvalidatedDynamicMethodCallGood4.js

```diff
@@ -0,0 +1,20 @@
+var express = require('express');
+var app = express();
+
+var actions = new Map();
+actions.put("play", function play(data) {
+    // ...
+});
+actions.put("pause", function pause(data) {
+    // ...
+});
+
+app.get('/perform/:action/:payload', function(req, res) {
+    if (typeof actions.get(req.params.action) === 'function') {
+        let action = actions.get(req.params.action); // OK but flagged [INCONSISTENCY]
+        // GOOD: `action` is either the `play` or the `pause` function from above
+        res.end(action(req.params.payload));
```

Comment:
```suggestion
        let action = actions.get(req.params.action);
        res.end(action(req.params.payload));  // OK but flagged [INCONSISTENCY]. `action` is either the `play` or the `pause` function from above
```

We usually put the `//OK ` or `//NOT OK` comments on the line that is flagged.  
For path-queries it's the sink that gets flagged, so the comment should go there.